include/cuco/detail/equal_wrapper.cuh

```diff
@@ -78,9 +78,9 @@ struct equal_wrapper {
   /**
    * @brief Order-sensitive equality operator.
    *
-   * @note This function always compares the left-hand side element against sentinel values first
+   * @note This function always compares the right-hand side element against sentinel values first
    * then performs a equality check with the given `equal_` callable, i.e., `equal_(lhs, rhs)`.
-   * @note Container (like set or map) keys MUST be always on the left-hand side.
+   * @note Container (like set or map) buckets MUST be always on the right-hand side.
```

Comment:
buckets? slots?

Reply:
you are the expert here :)
Is bucket a more proper term than slot in this case?

Reply:
Open-addressing means buckets consist of a single table slot, so it's interchangeable. Let's leave it as is.